modules/update-upgrading-web.adoc

```diff
@@ -19,17 +27,41 @@ of the Customer Portal.
 
 . From the web console, click *Administration* > *Cluster Settings* and review
 the contents of the *Overview* tab.
-.. For production clusters, ensure that the *CHANNEL* is set to `stable-4.2`.
+. For production clusters, ensure that the *CHANNEL* is set to the correct channel for
+ifdef::within[]
+the version that you want to update to,
+endif::within[]
+ifdef::between[]
+your current minor version,
+endif::between[]
+such as `stable-{product-version}`
 +
 [IMPORTANT]
 ====
-For production clusters, you must subscribe to the `stable-4.2` channel.
+For production clusters, you must subscribe to a `stable-\*` or `fast-*` channel.
 ====
-.. If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
+** If the *UPDATE STATUS* is not *Updates Available*, you cannot upgrade your
 cluster.
-.. The *DESIRED VERSION* indicates the cluster version that your cluster is running
+** The *DESIRED VERSION* indicates the cluster version that your cluster is running
 or is updating to.
 
-. Click *Updates Available*, select a version to update to, and click *Update*.
+. Click *Updates Available*, select
+ifdef::within[]
+a version to update to,
+endif::within[]
+ifdef::between[]
+the highest available version
+endif::between[]
+and click *Update*.
 The *UPDATE STATUS* changes to `Updating`, and you can review the progress of
 the Operator upgrades on the *Cluster Operators* tab.
+
+ifdef::between[]
+. After the update completes and the Cluster Version Operator refreshes the available updates, check if more updates are available in your current channel.
+--
+** If updates are available, continue to perform updates in the current channel until you can no longer update.
+** If no updates are available, change the *CHANNEL* to the `stable-*` or `fast-*` channel for the next minor version, and update to the version that you want in that channel.
```

Comment:
@kalexand-rh - the build is failing here because it is getting confused with the multiple * on the same line (it's an old known bug).

Suggest overriding our guidelines to not have backticks for 'stable-* or fast-*' bit.